konfera/models/talk.py

```diff
@@ -23,13 +24,22 @@
 
 class Talk(models.Model):
     title = models.CharField(max_length=256)
-    abstract = models.TextField()
+    abstract = models.TextField(help_text=_('Abstract will be published in the schedule.'))
     type = models.CharField(choices=TALK_TYPE, max_length=32)
     status = models.CharField(choices=TALK_STATUS, max_length=32)
-    duration = models.IntegerField(choices=TALK_DURATION)
-    primary_speaker = models.ForeignKey('Speaker', related_name='secondary_speaker')
-    secondary_speaker = models.ForeignKey('Speaker', related_name='primary_speaker', blank=True, null=True)
+    duration = models.IntegerField(choices=TALK_DURATION, help_text=_('Talk duration in minutes.'))
+    primary_speaker = models.ForeignKey('Speaker', related_name='primary_speaker_talks')
+    secondary_speaker = models.ForeignKey('Speaker', related_name='secondary_speaker_talks', blank=True, null=True)
     event = models.ForeignKey('Event')
 
     def __str__(self):
         return self.title
+
+    def clean(self):
+        if hasattr(self, 'primary_speaker') and hasattr(self, 'secondary_speaker') \
+                and self.primary_speaker == self.secondary_speaker:
+            msg = _('%(primary)s have to be different than %(secondary)s') % {
```

Comment:
"primary speaker have to be different than secondary speaker"

- add a period to the end
- capitalize the **P**rimary speaker